src/style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: rgb(237, 237, 237);
+}
+
+section {
+  background-color: #fff;
+  width: 90%;
+  box-shadow: 1px 3px 8px 7px rgb(186, 185, 185);
+  margin-top: 50px;
+}
+
+h1 {
+  font-family: cursive;
+  font-weight: 500;
+  margin-top: 10px;
+}
+
+.add-todo {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 2px solid rgb(199, 195, 195);
+  margin-bottom: 8px;
+  height: 50px;
+}
+
+.refresh-icon {
+  display: flex;
+  align-items: center;
+  margin-right: 1%;
+}
+
+.list {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  border-bottom: 2px solid rgb(199, 195, 195);
+}
+
+.inputt {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 8px;
+  margin-bottom: -10px;
+  border-bottom: 2px solid rgb(199, 195, 195);
+}
+
+img {
+  width: 16px;
+  height: 16px;
+  margin-right: 1%;
+  margin-top: 8px;
+}
+
+.inputt img {
+  margin-top: 15px;
+}
+
+img:hover {
+  cursor: all-scroll;
+}
+
+#enter-todo {
+  border: none;
+  outline: none;
+  color: gray;
+  font-family: cursive;
+  font-size: large;
+  padding-top: 9px;
+  padding-bottom: 9px;
+}
+
+.list-item {
+  font-family: cursive;
+  list-style: none;
+  margin-left: -40px;
+  padding: 5px 0 5px 0;
+  color: rgb(79, 76, 76);
+}
+
+#complete {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: cursive;
+  font-size: large;
+  background-color: rgb(218, 217, 217);
+  height: 60px;
+  margin-top: -22px;
+}
+
+#error {
+  color: red;
+  font-family: cursive;
+  font-weight: 600;
+}
+
+ul.active {
+  display: none;
+}
+
+.image {
+  display: flex;
+}
+
+.remove {
+  display: none;
+}
```

Comment:
Sorry for the confusion this is the part I wrote about.

Reply:
Alright, got it. Thank you. I will definitely implement it.